### DISCUSSION TITLE
I don't know what the -0300 at the right of the core compilation date means.

### DISCUSSION THREAD

**Discussion by quilumipe2** [azerothcore/azerothcore-wotlk#8971] [Offtopic]
Greetings, I just made a fresh compile of AzerothCore emulator and the .server info command shows the following:  AzerothCore rev. d80f7d467d47+ 2021-11-05 07:47:08 -0300 (master branch) (Unix, Release, Static), I'm curious of what that "-0300" number means. Thankyou!

I'm guessing it has something to do with CMake, but I'm not sure

**Comment by nicoaravena** [comment]
Is the timezone, -0300 (utc-3) is somewhere in Latam, like America/Santiago.

**Reply by quilumipe2** [reply]
dammit, I live in Spain, could that cause troubles?
